pages/funnel-analytics/with-async-table.page.tsx

```diff
@@ -0,0 +1,188 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
nit: there is a few features here that are not used in tests, e.g. inline-editing labels, preferences, sorting and filtering.
are they going to be used in the future? or can they be removed for the sake of simplicity